tests/Unit/inc/functions/getRocketCacheRejectUri.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace WP_Rocket\Tests\Unit\inc\functions;
+
+use stdClass;
+use Brain\Monkey\Functions;
+use WPMedia\PHPUnit\Unit\TestCase;
+
+/**
+ * @covers ::get_rocket_cache_reject_uri
+ * @group Functions
+ * @group Options
+ */
+class Test_GetRocketCacheRejectUri extends TestCase {
+	/**
+	 * @dataProvider providerTestData
+	 */
+	public function testShouldGetRocketCacheRejectUri( $config, $expected ) {
+		$GLOBALS['wp_rewrite']            = new stdClass();
+		$GLOBALS['wp_rewrite']->feed_base = 'feed/';
```

Comment:
To be sure we have no conflicts between tests, add a `tearDown()` method, which will call `unset( $GLOBALS['wp_rewrite'] )`